examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -225,30 +333,34 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
     function setInfoControlCountMap() {
       // Interaction function
       function highlightFeature(leafletEvent) {
-        var layer = leafletEvent.target;
-        layer.setStyle($scope.styles.hoverStyle);
-        if (!L.Browser.ie && !L.Browser.opera) {
-          layer.bringToFront();
+        if (cloudberry.parameters.maptype == 'countmap'){
+          var layer = leafletEvent.target;
+          layer.setStyle($scope.styles.hoverStyle);
+          if (!L.Browser.ie && !L.Browser.opera) {
+            layer.bringToFront();
+          }
+          $scope.selectedPlace = layer.feature;
         }
-        $scope.selectedPlace = layer.feature;
       }
 
       function resetHighlight(leafletEvent) {
-        var style;
-        if (!$scope.status.init)
-          style = {
-            weight: 1.5,
-            fillOpacity: 0.5,
-            color: '#92d1e1'
-          };
-        else
-          style = {
-            weight: 1.5,
-            fillOpacity: 0.5,
-            color: '#92d1e1'
-          };
-        if (leafletEvent)
-          leafletEvent.target.setStyle(style);
+        if (cloudberry.parameters.maptype == 'countmap'){
+          var style;
+          if (!$scope.status.init)
```

Comment:
Let's use brackets {} for if and else to remove any confusion.

Reply:
Okay, I will fix it. I didn't write that part though.